packages/xarc-tag-renderer/src/render-execute.ts

```diff
@@ -23,7 +23,7 @@ const {
   STEP_SUB_TEMPLATE
 } = executeSteps;
 
-function handleSubTemplate(tkId: string, step, result: any, xt: any, cb: Function) {
+function handleSubTemplate(tkId: string, step, result: any, xt: any, cb: any) {
```

Comment:
if we don't want to use Function (okay, it's frowned upon) .. lets define it a bit more

```ts
type callback = (...args: any) => void;
function handleSubTemplate(tkId: string, step, result: any, xt: any, cb: callback) {
```
or inline...
```ts
function handleSubTemplate(tkId: string, step, result: any, xt: any, cb: (...args: any) => void) {
```

also - could add some return types to the function's signature ..